x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/mapping_info_panel/mapping_info_panel.tsx

```diff
@@ -276,12 +277,12 @@ export class MappingInfoPanel extends Component<Props, State> {
         >
           <EuiSwitch
             name={'enabled'}
-            label={
-              <FormattedMessage
-                id="xpack.security.management.editRoleMapping.roleMappingEnabledLabel"
-                defaultMessage="Enable mapping"
-              />
-            }
+            label={i18n.translate(
```

Comment:
**note** This just resolves an EUI warning where `EuiSwitch`'s `label` must be a string when `showLabel={false}`.